dags/springer/parser.py

```diff
@@ -110,6 +112,15 @@ def __init__(self) -> None:
         super().__init__(extractors)
 
     def _get_abstract(self, article: ET.Element):
+        def is_latex(text):
+            return (
+                re.search(r"\\\[(.+?)\\\]", text)
+                or re.search(r"\\\((.+?)\\\)", text)
+                or re.search(r"\\\[(.+?)\\\]", text)
+                or re.search(r"\\begin\{displaymath\}(.+?)\\end\{displaymath\}", text)
+                or re.search(r"\$\$(.+)\$\$", text)
```

Comment:
latex not always are between `$` can we just get the value from `CDATA`?

Reply:
No because the LaTeX expressions are not always in CDATA. They seems to be between tags with `format="TEX"`. Anyways as discussed, I need to change this to remove the regex

Reply:
Latex expressions are now coming from `<EquationSource format="TEXT>` tags 👍